### ISSUE TITLE
Mouse-scroll: Zoom in/out "jumps" on /maps page 

### ISSUE THREAD

**Issue by filipefurtad0** [openfoodfoundation/openfoodnetwork#8637]
## Description

Zooming in/out with the mouse-wheel causes the center of the map to "jump" around and triggers some errors seen on the console.

This is was observed when zooming in/out with the +/- signs on the map.

## Expected Behavior

The map should remain centered on the cursor coordinates on the screen, while zooming.

## Actual Behaviour

While zooming with the mouse-wheel the map changes position erratically..

## Steps to Reproduce

1. Visit /maps
2. Open the console
3. Scroll around and till you notice some jumps
4. Notice the (probably related) console errors

## Animated Gif/Screenshot

![image](https://user-images.githubusercontent.com/49817236/146527369-3dc32282-5f57-4d04-9fe1-8f2487360415.png)

## Workaround

None needed.

## Severity

bug-s4: it's annoying, but you can use it

## Your Environment

* Version used: v4.1.11
* Browser name and version: Firefox 94
* Operating System and version (desktop or mobile): Ubuntu 20.04

**Comment by jibees** [comment]
Can't reproduce with https://openfoodnetwork.org.uk/map and Chrome_114.

Is this still available? (this bug is pretty old actually)

**Comment by filipefurtad0** [comment]
Yes, I can confirm this still happens on Firefox 115. But indeed not happening on Chrome.

**Comment by jibees** [comment]
Sorry, didn't read _Browser name and version: Firefox 94_ ! 🙏 🤓

**Comment by jibees** [comment]
Can't reproduce on Firefox as well I guess: 

![ezgif-4-14906f4aaa](https://github.com/openfoodfoundation/openfoodnetwork/assets/296452/b0a11730-ce1a-40bb-97c3-35d570c15788)

**Comment by filipefurtad0** [comment]
> Can't reproduce on Firefox as well I guess:

:eyes: 

On Firefox 115, on Ubuntu 22.04:

https://www.loom.com/share/27ee9ae7c57c4c2b8fbbb45c35bee638?sid=83d811e4-814f-4c2b-a42c-41ad65cddb85

But I'm guessing this affects other systems as well? (I have not investigated which ones)

**Comment by drummer83** [comment]
When I observed this, I had a feeling that it could be linked to enterprises being located far away. E.g. the German instance has producers located in Spain. Before we had those, my feeling was that the map was working well.
It's just a wild guess though!

**Comment by NiftyGaloot** [comment]
An update on this issue: I am unable to reproduce the issue with:
Firefox 152.0.1 on Ubuntu 24.04.4 LTS or Linux Mint 21.3 Cinnamon
